modules/ROOT/pages/as2/as2-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,249 @@
+= AS2 Connector - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Supported Upgrade Paths
+
+This a direct upgrade from the AS2 connector 4.x to 5.0.0.
```

Comment:
Please replace AS2 connector with the full connector name, as defined in the writing style guide.

Reply:
oops, fixed.